cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -353,6 +354,26 @@ func StartControllers(s *options.CMServer, kubeClient *client.Client, kubeconfig
 		serviceaccountcontroller.DefaultServiceAccountsControllerOptions(),
 	).Run()
 
+	if s.HealthCheckControllerSyncPeriod > 0 {
+		config := client.AddUserAgent(kubeconfig, "healthchecker-controller")
+
+		// NB: in order for this health checker to work, the controller manager must be
+		// able to reach all of the API servers listed in the "default/kubernetes"
+		// endpoints.
+		checkerTransport, err := client.TransportFor(config)
+		if err != nil {
+			panic(err)
```

Comment:
@smarterclayton ACK.  I was just following the style of the `OrDie` commands (since this is effectively `NewHTTPClientOrDie`) but `glog.Fatalf` makes sense too.